wpilibcExamples/src/main/cpp/examples/SysId/include/Constants.h

```diff
@@ -0,0 +1,81 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+#pragma once
+
+#include <array>
+#include <numbers>
+
+#include <units/angle.h>
+#include <units/angular_velocity.h>
+#include <units/length.h>
+#include <units/time.h>
+#include <units/voltage.h>
+
+namespace constants {
+namespace drive {
+static constexpr int kLeftMotor1Port = 0;

```

Comment:
These should all be `inline constexpr` instead of `static constexpr` to avoid ODR violations.